x/qgb/types/abi_consts.go

```diff
@@ -0,0 +1,170 @@
+	// https://github.com/OpenZeppelin/openzeppelin-contracts/blob/c9f328ef66251db7fac7c704dd6c5523fc53b0ab/contracts/cryptography/ECDSA.sol#L69-L82 //nolint:lll
+	EthSignPrefix = "\x19Ethereum Signed Message:\n32"
+
+	// InternalQGBabiJSON is the json encoded abi for private functions in the
+	// qgb contract. This is needed to encode data that is signed over in a way
+	// that the contracts can easily verify.
+	InternalQGBabiJSON = `[
+			{
+			"inputs": [
+				{
+				"components": [
+					{
+					"internalType": "address",
+					"name": "addr",
+					"type": "address"
+					},
+					{
+					"internalType": "uint256",
+					"name": "power",
+					"type": "uint256"
+					}
+				],
+				"internalType": "struct Validator[]",
+				"name": "_validators",
+				"type": "tuple[]"
+				}
+			],
+			"name": "computeValidatorSetHash",
+			"outputs": [
+				{
+				"internalType": "bytes32",
+				"name": "",
+				"type": "bytes32"
+				}
+			],
+			"stateMutability": "pure",
+			"type": "function"
+			},
+			{
+			"inputs": [
+				{
+				"internalType": "bytes32",
+				"name": "_bridge_id",
+				"type": "bytes32"
+				},
+				{
+				"internalType": "bytes32",
+				"name": "_separator",
+				"type": "bytes32"
+				},
+				{
+				"internalType": "uint256",
+				"name": "_nonce",
+				"type": "uint256"
+				},
+				{
+				"internalType": "bytes32",
+				"name": "_dataRootTupleRoot",
+				"type": "bytes32"
+				}
+			],
+			"name": "domainSeparateDataRootTupleRoot",
+			"outputs": [
+				{
+				"internalType": "bytes32",
+				"name": "",
+				"type": "bytes32"
+				}
+			],
+			"stateMutability": "pure",
+			"type": "function"
+			},
+			{
+			"inputs": [
+				{
+				"internalType": "bytes32",
+				"name": "_bridge_id",
+				"type": "bytes32"
+				},
+				{
+				"internalType": "bytes32",
+				"name": "_separator",
+				"type": "bytes32"
+				},
+				{
+				"internalType": "uint256",
+				"name": "_nonce",
+				"type": "uint256"
+				},
+				{
+				"internalType": "uint256",
+				"name": "_powerThreshold",
+				"type": "uint256"
+				},
+				{
+				"internalType": "bytes32",
+				"name": "_validatorSetHash",
+				"type": "bytes32"
+				}
+			],
+			"name": "domainSeparateValidatorSetHash",
+			"outputs": [
+				{
+				"internalType": "bytes32",
+				"name": "",
+				"type": "bytes32"
+				}
+			],
+			"stateMutability": "pure",
+			"type": "function"
+			}
+		]`
+
+	// Domain separator constants copied directly from the contracts
+	ValidatorSetDomainSeparator   = "0x636865636b706f696e7400000000000000000000000000000000000000000000"
+	DataCommitmentDomainSeparator = "0x7472616e73616374696f6e426174636800000000000000000000000000000000"
+)
+
+var (
+	ExternalQGBabi     abi.ABI
+	InternalQGBabi     abi.ABI
+	BridgeValidatorAbi abi.Arguments
+
+	VsDomainSeparator ethcmn.Hash
+	DcDomainSeparator ethcmn.Hash
+)
+
+func init() {
+	contractAbi, err := abi.JSON(strings.NewReader(wrapper.QuantumGravityBridgeMetaData.ABI))
```

Comment:
Any idea why this is empty: 
https://github.com/celestiaorg/quantum-gravity-bridge/blob/6b6efce45c4ba67cf277d5e34c11c33e15084b25/ethereum/solidity/wrappers/QuantumGravityBridge.sol/wrapper.go#L59
?

Reply:
Aren't we referencing this instead? 

https://github.com/celestiaorg/quantum-gravity-bridge/blob/6b6efce45c4ba67cf277d5e34c11c33e15084b25/ethereum/solidity/wrappers/QuantumGravityBridge.sol/wrapper.go#L765-L767